compiler/src/iree/compiler/Dialect/HAL/Target/WebGPU/WebGPUTarget.cpp

```diff
@@ -84,7 +84,7 @@ class WebGPUTargetBackend : public TargetBackend {
   }
 
   void buildTranslationPassPipeline(OpPassManager &passManager) override {
-    buildSPIRVCodegenPassPipeline(passManager);
+    buildSPIRVCodegenPassPipeline(passManager, /*enableFastMath=*/true);
```

Comment:
Can you add a comment here with some context for why WebGPU wants fast math? You could link to https://github.com/iree-org/iree/issues/10142, some gpuweb issues, and/or the webgpu spec.

Reply:
Yup, added comments there.